www/modules/recalls/views/Search.html

```diff
@@ -1,17 +1,35 @@
 <div class="recalls search" layout="column" layout-fill>
-    <div class="backdrop">
-        <div class="search-results-feedback" layout-fill
-             layout="column" layout-align="center center"
-             ng-if="search.recalls.length == 0">
-            <div i18n="recalls.search.noResults"></div>
-            <md-button scan-button class="md-primary md-raised" i18n="recalls.search.scanButton"></md-button>
-        </div>
+    <div class="backdrop" layout="column" layout-align="center center" layout-fill>
+        <div class="search-results-feedback" i18n="recalls.search.noResults"
+             ng-if="search.recalls.length == 0"></div>
+        <form name="searchForm" ng-submit="search.doSearch()"
+              layout="column" layout-align="center center"
+              ng-if="!search.recalls.length">
+            <md-input-container style="width:100%">
+                <label>Barcode/UPC</label>
```

Comment:
This should be be in the translation bundle and lets change the text to "Keywords or UPC"